src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+
+type Users = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  userId: number;
+};
+
+type TodoInfoProps = {
+  todo: Todo;
+  users: Users[];
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo, users }) => {
+  return (
+    <article data-id={todo.id} className="TodoInfo TodoInfo--completed">
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+      {users.filter((user) => todo.userId === user.id).map((user) => (
```

Comment:
You don't need to pass all users to the TodoInfo. You can find it in the list component instead. BTW use the `find` method instead